unity-renderer/Assets/UIComponents/Scripts/Components/SearchBar/SearchBarComponentModel.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+
+[Serializable]
+public class SearchBarComponentModel : BaseComponentModel
+{
+    public float idleTimeToTriggerSearch = 1f;
```

Comment:
To me 1 second felt a bit too long

Reply:
its the default value for all the search bars in the ui